src/main.py

```diff
@@ -1,8 +1,15 @@
-# Resolve the problem!!
+import re
 
 
 def run():
     # Start coding here
+    text = ''
+    with open('./src/encoded.txt', 'r', encoding='utf-8') as f:
+        text = f.read()
+
+    message = ''.join(re.findall(r'\w+(?a)', text))
```

Comment:
Es mejor compilar tu expresión regular antes de usarla.

Reply:
En python3.8 aparece:
src/main.py:10: DeprecationWarning: Flags not at the start of the expression '\\w+(?a)'
  message = ''.join(re.findall(r'\w+(?a)', text))

Reply:
Estuve leyendo y parece que es un bug 🐞

https://bugs.python.org/issue22493

Reply:
https://platzi.com/clases/1301-expresiones-regulares/11879-python9865/

Reply:
Te comentaba que al parecer es un bug por que intenté compilando la expresión regular y lanzaba el mismo _DeprecationWarning_

Sin compilar:
`src/main.py:10: DeprecationWarning: Flags not at the start of the expression '\\w+(?a)'
  message = ''.join(re.findall(r'\w+(?a)', text))
`

Compilando:
`
src/main.py:7: DeprecationWarning: Flags not at the start of the expression '\\w+(?a)'
  regex = re.compile(r'\w+(?a)')
`

So... intente otra solución.

Reply:
creo que no tienes que usar `r` al inicio. creo que eso es lo que ya no van a soportar. El compilador te permite hacer la expresión regular si poner el flag `r`

Reply:
Encontré esta otra guía
https://platzi.com/blog/expresiones-regulares-python/